src/Analyzers/CSharp/Analyzers/SimplifyLinqExpression/CSharpSimplifyLinqExpressionDiagnosticAnalyzer.cs

```diff
@@ -55,16 +56,16 @@ private void OnCompilationStart(CompilationStartAnalysisContext context)
             var whereMethods = ImmutableHashSet.CreateBuilder<IMethodSymbol>();
             var linqMethods = ImmutableHashSet.CreateBuilder<IMethodSymbol>();
 
-            var enumFullName = typeof(Enumerable).FullName ?? "System.Linq.Enumerable";
-            var enumNamedType = context.Compilation.GetTypeByMetadataName(enumFullName);
+#pragma warning disable CS8604 // Possible null reference argument.
+            var enumNamedType = context.Compilation.GetTypeByMetadataName(typeof(Enumerable).FullName);
```

Comment:
We should instead We should instead define an extension method for getting well known types, like https://github.com/dotnet/roslyn/blob/a2e9350d071a2343ed23b78760a4cdeefb167bfd/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ICompilationExtensions.cs#L121-L122